dev-libs/libowfat/libowfat-0.31-r1.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+inherit flag-o-matic toolchain-funcs eutils
+
+DESCRIPTION="reimplement libdjb - excellent libraries from Dan Bernstein"
```

Comment:
DESCRIPTION could be less advertising and more descriptive.

Reply:
Suggest something…

Reply:
you could describe what the lib is good for instead rating it ```excellent``` I do not think that the authors name is important here.

Reply:
Honestly I think that is a task for the package maintainer, as it should be consistently be changed in every ebuild at the same time.